experimental/FTheoryTools/src/LiteratureModels/constructors.jl

```diff
@@ -525,7 +525,7 @@ function _construct_literature_model_over_arbitrary_base(model_dict::Dict{String
   elseif model_dict["model_descriptors"]["type"] == "hypersurface"
 
     # Extract base variable names
-    auxiliary_base_vars = [string(g) for g in gens(auxiliary_base_ring)]
+    auxiliary_base_vars = [string(g) for g in symbols(auxiliary_base_ring)]
 
     # Extract fiber ambient space
     rays = [[a for a in b] for b in model_dict["model_data"]["fiber_ambient_space_rays"]]
```

Comment:
Off-topic for this PR, but: I wonder if `[a for a in b]` shouldn't be replaced by `collect(b)` here and elsewhere? This line could perhaps even become
```suggestion
    rays = collect.(model_dict["model_data"]["fiber_ambient_space_rays"])
```
(Or maybe not, I did not try it and depending on what the code does this may or may not work, and it may or may not affect performance and allocations in either a good or bad way). So, just something to be potentially aware of, @HereAround